bowtie/schemas/public-implementations.json

```diff
@@ -0,0 +1,77 @@
+{
+  "title": "Public Implementations",
```

Comment:
You're focused a bit too much on the word "public". Public simply means here' we're really providing an API for others to consume, as opposed to just something we (Bowtie, the people who work on Bowtie) use ourselves.

But "public" shouldn't really appear anywhere. Here let's put this file in `bowtie/schemas/api/v1/json-schema-org/implementations.json`.